### DISCUSSION TITLE
Remote control of Osmand

### DISCUSSION THREAD

**Discussion by bandito40** [osmandapp/OsmAnd#14815] [General]
Yesterday I was recording my travels. I dropped my phone and the screen broke. The phone is still active as I can hear the notification everytime I get a message.
Is there a way I can get it to stop and save the recording? I do have it set to record to a sdcard so I can pull that out afterward.

**Comment by vshcherb** [comment]
I think it's not possible actually. You need to get access to screen or you can restart the app via Android Debug Tools and it will save track automatically

**Reply by bandito40** [reply]
Would if be the same if I just hold the power button to shut it off then take out the sdcard afterwards?
